test-suite/src/function/take.rs

```diff
@@ -0,0 +1,61 @@
+use {crate::*, gluesql_core::executor::EvaluateError, gluesql_core::prelude::Value::*};
+
```

Comment:
You can consider a few more tests which handles `NULL`
either `expr` or `size` field is null, then the result is null in sql evaluation.
e.g.
```sql
TAKE(NULL, 0)
-- NULL

TAKE(items, NULL)
-- NULL
```

Reply:
Null has more high priority than size or list validation?

e.g.
```rust
TAKE(NULL, -5)
-- NULL? or FunctionRequiresUSizeValue?
```

Reply:
that's really a good point.
for now, we don't have strict policy about the priority on what to handle earlier, yet.
it may depend on the arrange of implementation codes so it's all up to you.

Reply:
I think about that size or list validation has more priority,
I have to choose one more option to give higher priority validation between them.

So I adapt the way for just returning NULL when NULL value exists in query.